samples/04_gis_analysts_data_scientists/forecasting_air_temperature_in_california.ipynb

```diff
@@ -0,0 +1,2703 @@
+{
```

Comment:
<p>Next, the dataset is prepared wherein the variables of station, possible sunshine, snow cover, maximum temperature, minimum temperature, total sunshine, and peak wind gust speed are dropped, and then the dataset is chosen from 1987 onwards, to avoid missing values.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>